ci-operator/config/openshift/cluster-config-operator/openshift-cluster-config-operator-release-4.4.yaml

```diff
@@ -8,7 +8,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.13
+    tag: golang-1.12
```

Comment:
isn't 4.4 1.13?

Reply:
nope https://github.com/openshift/cluster-policy-controller/blob/release-4.4/Dockerfile.rhel

Reply:
it is https://github.com/kubernetes/kubernetes/blob/release-1.17/hack/lib/golang.sh#L470

Reply:
the units are built from go 1.13 and that go 1.13 is injected into the image builds as go 1.12 so images are build with 1.13 no matter what the dockerfile says. We can't lower this to 1.12 - that would actually run the units and image builds with the wrong version - the dockefile needs to be fixed first and then the replacement here bellow - only that way we keep building with correct 1.13 all the time

Reply:
You're right, fixed in https://github.com/openshift/cluster-policy-controller/pull/31

Reply:
and here as well